lib/modules/dosomething/dosomething_signup/includes/SignupsController.php

```diff
@@ -1,73 +1,59 @@
 <?php
 
-/**
- * @file
- * Provides classes for the Signup Entity.
- */
+class SignupsController extends EntityAPIController {
```

Comment:
Ah, the Dream! I defs like the sound of that. Gonna make an issue from it for future improvements 💃